config/routes.rb

```diff
@@ -78,13 +78,20 @@
 
     end
 
-    resources :inventories, only: [:create, :index, :show] do
+    resources :inventories, only: [:create, :index, :show, :update] do
       resources :invitations, controller: 'inventory_invitations', only: [:create]
       resources :access_requests, controller: 'inventory_access_requests', only: [:index, :create, :update]
       resource :permissions, controller: 'inventory_permissions', only: [:show, :update]
-      resources :participants, controller: 'inventory_participants', only: [:create, :destroy]
+      resources :participants, controller: 'inventory_participants', only: [:create, :destroy, :index]
+        get 'participants/all', to: 'inventory_participants#all'
       resources :invitables, controller: 'inventory_invitables', only: [:index]
       resources :product_entries, only: [:index, :show, :create, :update]
+
+      resources :learning_questions, only: [:index, :create, :update, :destroy] do
+        collection do
+          get 'exists', to: 'learning_questions#exists'
```

Comment:
It's a smaller payload - all I want to verify is if the entity exists before I do anything else with it.  It's used for determining if we wish to open the modal for a user to encourage them to create learning questions.  It can be found here:  https://github.com/MobilityLabs/pdredesign-server/blob/staging/app/assets/javascripts/client/controllers/responses/ResponseCtrl.js

I don't use the `show` since I don't want to bring back more data than I care about on the request.  All I need is the 200 and if I can save the bandwidth, I'd like to.